src/coreclr/jit/emitxarch.cpp

```diff
@@ -1456,7 +1642,18 @@ bool emitter::TakesRex2Prefix(const instrDesc* id) const
 emitter::code_t emitter::AddEvexPrefix(const instrDesc* id, code_t code, emitAttr attr)
 {
     // Only AVX512 instructions require EVEX prefix
-    assert(IsEvexEncodableInstruction(id->idIns()));
+    // After APX, some instructions in legacy or vex space will be promoted to EVEX.
```

Comment:
```suggestion
    // After APX, some instructions in legacy or VEX space will be promoted to EVEX.
```
I'd prefer `VEX` to always capitalized (or, if in a function name with "CamelCasing", `Vex`). After all "vex" is an English word...